libfuse/folderlist.go

```diff
@@ -182,7 +183,7 @@ func (fl *FolderList) Lookup(ctx context.Context, req *fuse.LookupRequest, resp
 		return nil, fuse.ENOENT
 	}
 
-	h, err := libkbfs.ParseTlfHandlePreferred(
+	h, err := libfs.ParseTlfHandlePreferredQuick(
```

Comment:
What if we had an RPC to resolve many handles in one call, and then cache them in KBFS for later?

Reply:
Sure, if the service gave us something like that it might make things a bit more efficient.  Though if we used it by default on an `ls`, it might be risky for people with tons of favorites who could otherwise use bare `/bin/ls` to avoid the overhead of resolving all their favorites.